meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -440,6 +440,8 @@ EFI_PROVIDER:imx8mp-lpddr4-evk-ebbr ?= "grub-efi"
 OSTREE_BOOTLOADER:imx8mp-lpddr4-evk-ebbr ?= "grub"
 IMAGE_EFI_BOOT_FILES:sota:imx8mp-lpddr4-evk-ebbr ?= "${@make_efi_dtb_boot_files(d)}"
 WKS_FILE:sota:imx8mp-lpddr4-evk-ebbr ?= "efidisk-sota.wks"
+# imx8mp comes with an AzureWave CM276MA PCIe Card
+MACHINE_FEATURES:append:imx8mm-lpddr4-evk = " nxp89xx mxm-mwifiex-load"
```

Comment:
Should be imx8mp instead of imx8mm.

Reply:
Great catch.  I fixed this on my build box and didn't re-sync back to my local files.

Reply:
Fixed